### PR TITLE
modify provision_vm_test to fit eco vcenter CI

### PR DESCRIPTION
**Pull request by shellymiron**
Following this [Jira](https://issues.redhat.com/browse/ACA-1554) - this is a PR to add the provision_vm integration test to the eco-vcenter-ci action.

### CODE REVIEW

**Comment by shellymiron** [comment]
@elsapassaro please review :)

File: tests/integration/targets/.DS_Store
Comment (mikemorency) [inline tests/integration/targets/.DS_Store:1]: we should ignore these in the `.gitignore`

Comment (mikemorency) [reply tests/integration/targets/.DS_Store:1]: we should also add the `integration_config.yml` file to the gitignore

File: tests/integration/targets/vmware_ops_provision_vm_test/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      datacenter: "{{ provision_vm_datacenter }}"
+
+  - name: Create a VM from the template
+    loop: "{{ provision_vms_from_template }}"
+    ansible.builtin.include_tasks: provision_vms_with_validation.yml
+
+  - name: Update one the provisioned VMs
+    ansible.builtin.include_tasks: update_vm.yml
+
+  always:
```
Comment (elsapassaro) [inline tests/integration/targets/vmware_ops_provision_vm_test/tasks/main.yml:32]: Here we should add a task to remove the network created in [this task](https://github.com/redhat-cop/cloud.vmware_ops/pull/53/files#diff-308cb569dd900a593a02620bce7b644c2ed748db334d802f61cc16492116022eR29-R37)

Comment (bardielle) [reply tests/integration/targets/vmware_ops_provision_vm_test/tasks/main.yml:32]: Will it be part of that PR? 
If so please change the PR to [WIP]

File: tests/integration/targets/.DS_Store
Comment (shellymiron) [reply tests/integration/targets/.DS_Store:1]: done

File: tests/integration/targets/vmware_ops_provision_vm_test/tasks/main.yml (hunk 1 above)
Comment (shellymiron) [reply tests/integration/targets/vmware_ops_provision_vm_test/tasks/main.yml:32]: tnx Elsa! I haven't noticed the missing cleanup here
I added the task, if the CI pass, we can merge this PR

**Review by elsapassaro — APPROVED** [review]

**Pull request merged**